gnosis/safe-react
# feat: add on-/off-chain tx event tracking

## What it solves
Transaction tracking

## How this PR fixes it
On-/off-chain interactions now emit a tracking event if they are successful.

## How to test it
- Sign off-chain and observe a tracking event.
- Sign/execute on-chain and observe a tracking event.

## Analytics changes
![image](https://user-images.githubusercontent.com/20442784/160584771-c6495f86-d041-433c-ad45-e50464e8e939.png)
